bpf/lib/lb.h

```diff
@@ -1172,8 +1178,14 @@ lb4_select_backend_id(struct __ctx_buff *ctx __maybe_unused,
 	if (unlikely(!backend_ids))
 		return 0;
 
-	index = hash_from_tuple_v4(tuple) % LB_MAGLEV_LUT_SIZE;
-	return map_array_get_16(backend_ids, index, (LB_MAGLEV_LUT_SIZE - 1) << 1);
```

Comment:
The `map_array_get_16` was added due to verifier complexity errors. Did you not run into them anymore?